sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -60,6 +60,19 @@ public boolean isEnabled() {
 
     options.getLogger().log(SentryLevel.DEBUG, "Capturing event: %s", event.getEventId());
 
+    SentryId sentryId = SentryId.EMPTY_ID;
+
+    if (event.getThrowable() != null
```

Comment:
this should be probably just before the `executeBeforeSend` callback, because we don't want to lose the chance to mark our session as crashed or something, it'd be the same concept of the `beforeSend` actualy

Reply:
I guess it should be aligned with other SDKs. I think ignoring means also doing as little as possible - in this case skipping all event processing - and not not marking session as crashed.

Reply:
@maciejwalkowiak it's a topic we've touched before, if you dropped an event in your beforeSend, you decided to do it, still, we prefer to give proper stats about your sessions.
the ignored exceptions are pretty much the same concept and I don't why it'd be different.
@bruno-garcia

Reply:
👍